src/content/en/fundamentals/building-components/examples/howto-checkbox.md

```diff
@@ -0,0 +1,616 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+
+{# wf_updated_on: 2017-08-14 #}
+{# wf_published_on: 2017-04-06 #}
+
+# HowTo: Components – howto-checkbox {: .page-title }
+
+{% include "web/_shared/contributors/ewagasperowicz.html" %}
+{% include "web/_shared/contributors/robdodson.html" %}
+{% include "web/_shared/contributors/surma.html" %}
+
+<link rel="stylesheet" href="main.css">
+
+## Summary {: #summary }
+
+A `<howto-checkbox` represents a boolean option in a form. The most common type
+of checkbox is a dual-type which allows the user to toggle between two
+choices -- checked and unchecked.
+
+The element attempts to self apply the attributes `role="checkbox"` and
+`tabindex="0"` when it is first created. The `role` attribute helps assistive
+technology like a screen reader tell the user what kind of control this is.
+The `tabindex` attribute opts the element into the tab order, making it keyboard
+focusable and operable. To learn more about these two topics, check out
+[What can ARIA do?][what-aria] and [Using tabindex][using-tabindex].
+
+When the checkbox is checked, it adds a `[checked]` boolean attribute, and sets
+a corresponding `checked` property to `true`. In addition, the element sets an
+`[aria-checked]` attribute to either `"true"` or `"false"`, depending on its
+state. Clicking on the checkbox with a mouse, or space bar, toggles these
+checked states.
+
+The checkbox also supports a `disabled` state. If either the `disabled` property
+is set to true or the `[disabled]` attribute is applied, the checkbox sets
```

Comment:
Removing them.

The reason we originally used them is because square brackets denote a CSS attribute selector. So if folks are familiar with that convention and see `[foo]` it helps to reinforce that we mean a foo="" attribute, instead of a js property. I think we started doing this in our element comments and it just kinda stuck. But i'm happy to undo it if it's not WF convention.